tests/fs-tree/fs-test.js

```diff
@@ -1014,8 +1014,14 @@ describe('FSTree fs abstraction', function() {
 
           let treeChanges = tree.changes();
           let newTreeChanges = newTree.changes();
+          expect(treeChanges).to.not.eql(newTreeChanges);
 
-          expect(treeChanges).to.eql(newTreeChanges);
+          expect(newTreeChanges).to.have.deep.property('0.0', 'create');
+          expect(newTreeChanges).to.have.deep.property('0.1', 'my-directory/subdir/ohai.txt');
```

Comment:
The second arg here should be a `cwd`-aware path (so in this case `ohai.txt`).  We should still leave `entry.relativePath` alone, so the assertion below is correct.